exercises/tournament/src/test/java/TournamentTest.java

```diff
@@ -19,92 +19,92 @@ public void justTheHeaderIfNoInput() {
     }
 
     @Test
-    @Ignore
+    @Ignore("Remove to run test")
```

Comment:
We would normally put the `@Ignore` tag above the `@Test` tag to be consistent with the other test files. It doesn't matter very much though :)

Reply:
Thanks for changing these! :) 👍